readme.md

```diff
@@ -0,0 +1,3 @@
+# Portfolio - trials microverse
+## This is a little project 
+This is an example of a page using HTML and CSS
```

Comment:
I kindly would recommend that you follow the structure of the microverse template. 🚀
You can find it here 👇 
https://github.com/microverseinc/readme-template

Reply:
Great job using a polite tone 👍 . I think it will be more helpful if you tell the student why he needs to use Microverse Template. Is it because of the project requirement or something else? 
This will allow the student to know why he is making changes 😉